.github/workflows/upload_binaries_to_release.yml

```diff
@@ -0,0 +1,54 @@
+name: Build release
+
+on:
+  release:
+    types: [created]
+
+permissions: write-all
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          #- windows-latest
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout tree
+        uses: actions/checkout@v4
+
+      - name: Set-up OCaml on Windows
+        uses: ocaml/setup-ocaml@v2
+        if: runner.os == 'Windows'
+        with:
+          ocaml-compiler: "ocaml-variants.4.07.1+mingw64"
+          opam-repositories: |
+            default: https://github.com/ocaml-opam/opam-repository-mingw.git#sunset
+    
+      - name: Set-up OCaml on Unix
+        uses: ocaml/setup-ocaml@v2
+        if: runner.os != 'Windows'
+        with:
+          ocaml-compiler: "5.1"
+          opam-repositories: |
+            default: https://github.com/ocaml/opam-repository.git
+
+      - run: opam install . --deps-only
+
+      - run: opam exec -- dune build
+
+      - run: mv ./_build/default/bin/main.exe ./_build/default/bin/Typocide_${{ runner.os }}.exe
```

Comment:
Can you change `Typocide_${{ runner.os }}.exe` to `typocide-${{ runner.os }}`. Eventually when we get a windows build, we can conditionally attach a `.exe` suffix to it for windows only.

Reply:
Okay, removed